extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -61,6 +61,15 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 			displayName: constants.edgeProjectTypeDisplayName,
 			description: constants.edgeProjectTypeDescription,
 			icon: IconPathHelper.sqlEdgeProject
+		},
+		{
+			id: constants.emptyAzureDbSqlDatabaseProjectTypeId,
+			projectFileExtension: constants.sqlprojExtension.replace(/\./g, ''),
+			displayName: constants.emptyAzureDbProjectTypeDisplayName,
+			description: constants.emptyAzureDbProjectTypeDescription,
+			icon: IconPathHelper.colorfulSqlProject,
```

Comment:
it would be nice to have a different icon for Azure sql db

Reply:
I'll ask for icon later